requests_kerberos/kerberos_.py

```diff
@@ -86,70 +84,75 @@ class HTTPKerberosAuth(AuthBase):
     object."""
     def __init__(
             self, mutual_authentication=REQUIRED,
-            service="HTTP", delegate=False):
+            service="HTTP", delegate=False, force_preemptive=False):
         self.context = {}
         self.mutual_authentication = mutual_authentication
         self.delegate = delegate
         self.pos = None
         self.service = service
+        self.force_preemptive = force_preemptive
 
-    def generate_request_header(self, response):
+    def generate_request_header(self, response, host, is_preemptive=False):
```

Comment:
If we're storing `force_preemptive` does this really need a parameter to the method? Couldn't the method just reference that attribute on the class?